css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+.header {
+  position: center;
+  padding-left: 415px;
+  font-size: 10px;
+  margin-bottom: 30px;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: #ed6761;
+}
+
+div.distance {
+  margin-top: 10px;
+  margin-left: 285px;
+}
+
+.input {
+  margin-left: 280px;
+  width: 420px;
+  border-radius: 7px;
+  border: 1px solid grey;
+  height: 30px;
+}
+
+.email {
+  font-size: x-small;
+  margin-bottom: 0;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.question {
+  font-size: x-small;
+  margin-bottom: 0;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
```

Comment:
These are doing the same thing, instead of repeating the ruleset, you should either combine them together or create a more generically named class that you can apply to both elements.